torch/csrc/inductor/aoti_torch/oss_proxy_executor.h

```diff
@@ -0,0 +1,99 @@
+#pragma once
+
+#include <ATen/core/dispatch/Dispatcher.h>
+#include <ATen/core/ivalue.h>
+#include <c10/macros/Export.h>
+#include <nlohmann/json.hpp>
+#include <torch/csrc/inductor/aoti_torch/c/shim.h>
+#include <torch/csrc/inductor/aoti_torch/proxy_executor.h>
+
+using json = nlohmann::json;
+
+namespace torch {
+namespace aot_inductor {
+
+enum DynamicArgType : int {
```

Comment:
Not to leak it in a global namespace
```suggestion
enum class DynamicArgType : int {
```